### PR TITLE
Set fixed session start time in test session

### PR DESCRIPTION
**Pull request by sopel39**
This simplifies writing time function tests.

### CODE REVIEW

File: presto-main/src/test/java/io/prestosql/SessionTestUtils.java [hunk 1]
```diff
@@ -15,16 +15,21 @@
...
 import static java.util.Arrays.stream;
 
 public final class SessionTestUtils
 {
+    public static final Instant SESSION_START_TIME = Instant.parse("2007-12-03T10:15:30.00Z");
+
     public static final Session TEST_SESSION = testSessionBuilder()
             .setCatalog("tpch")
             .setSchema(TINY_SCHEMA_NAME)
+            .setStartTime(SESSION_START_TIME.toEpochMilli())
```
Comment (findepi) [inline presto-main/src/test/java/io/prestosql/SessionTestUtils.java]: We either add this to `io.prestosql.testing.TestingSession#testSessionBuilder()` for all tests or add in specific tests only. 

Also, i don't know why `io.prestosql.SessionTestUtils#TEST_SESSION` is providing different defaults than `io.prestosql.testing.TestingSession#testSessionBuilder()`. Maybe we can remove `SessionTestUtils`?

Comment (sopel39) [reply presto-main/src/test/java/io/prestosql/SessionTestUtils.java]: done,

> Maybe we can remove `SessionTestUtils`?

This is probably a good idea, but not part of this PR

**Comment by sopel39** [comment]
ac

File: presto-main/src/main/java/io/prestosql/testing/TestingSession.java [hunk 2]
```diff
@@ -61,6 +63,7 @@ public static SessionBuilder testSessionBuilder()
     public static SessionBuilder testSessionBuilder(SessionPropertyManager sessionPropertyManager)
     {
         return Session.builder(sessionPropertyManager)
+                .setStartTime(SESSION_START_TIME.toEpochMilli())
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/testing/TestingSession.java]: move just before `.setTimeZoneKey()` as they are related.

File: presto-main/src/main/java/io/prestosql/testing/TestingSession.java [hunk 3]
```diff
@@ -40,6 +41,7 @@
 public final class TestingSession
 {
     public static final String TESTING_CATALOG = "testing_catalog";
+    public static final Instant SESSION_START_TIME = Instant.parse("2007-12-03T10:15:30.00Z");
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/testing/TestingSession.java]: - inline
- choose moment with fractional millis (eg `.123Z`)
- use some more recent year >=2017

**Comment by electrum** [comment]
What places need this other than `TestDateTimeFunctionsBase`?

**Comment by sopel39** [comment]
> What places need this other than `TestDateTimeFunctionsBase`?

We use that for customer-tailored functions, but I thought that it would be useful in master too

**Comment by dain** [comment]
Hey, @sopel39  and @findepi, can we come to a resolution on this?

**Comment by sopel39** [comment]
stalled and not needed. closing

**Pull request closed without merging**